delfi/distribution/Gamma.py

```diff
@@ -41,13 +41,14 @@ def std(self):
     @copy_ancestor_docstring
     def eval(self, x, ii=None, log=True):
         # univariate distribution only, i.e. ii=[0] in any case
-        return self._gamma.logpdf(x-self.offset) if log else self._gamma.pdf(x-self.offset)
+        res = self._gamma.logpdf(x-self.offset) if log else self._gamma.pdf(x-self.offset)
+        return res.squeeze()
```

Comment:
I don't think squeeze is a good idea, if x is only a single element it will return a 0D array. How about we call np.at_least2d at the beginning of the function, assert that the second dimension has  length 1, assert there's not more than 2 dims, then do an explicit reshape(-1) at the end.